data/domain/cardSets.tsx

```diff
@@ -77,6 +83,10 @@ export class CardSet {
         }
     }
 
+    static getMaximumCardSetLevel = (): number => {
+        return CardSetsPossibleLevels.reduce((a, b) => Math.max(a, b),-Infinity);
+    }
```

Comment:
I may not be understanding this implemention fully but it seems like a very complicated way of saying `return 7`

Is there anything else I'm missing here? it's just returning the highest number in an array of growing numbers.